components/channel/lib.rs

```diff
@@ -0,0 +1,154 @@
+extern crate crossbeam_channel;
+extern crate ipc_channel;
+extern crate serde;
+
+use crossbeam_channel::{Receiver as BaseReceiver, Sender as BaseSender};
```

Comment:
If you'd also like to reexport the macro, see how to go about that here:
https://github.com/crossbeam-rs/crossbeam/blob/02cc10efb351e4a6dfe4efd160dfc2169c9b455a/src/lib.rs#L74-L109

Reply:
thanks!